src/v2/Apps/Article/useArticleTracking.ts

```diff
@@ -0,0 +1,68 @@
+import { useTracking } from "react-tracking"
```

Comment:
@abhitip most of the tracking is here. See comments below.

Reply:
looks good!